.github/workflows/pytorch.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         pip install git+https://github.com/optuna/optuna.git
         python -c 'import optuna'
 
-        pip install -r pytorch/requirements.txt
+        SETUPTOOLS_USE_DISTUTILS=stdlib pip install -r pytorch/requirements.txt
```

Comment:
Could you add a TODO comment for this line?

Reply:
Thank you for your comment. I undo the last commit for consistency with the optuna repo's update.

Reply:
Let me merge this PR as it is now since it's fixing the currently broken daily CI. 🙇